includes/class-micropub-endpoint.php

```diff
@@ -117,14 +122,21 @@ protected static function check_scope( $scope ) {
 	/**
 	 * Parse the micropub request and render the document
 	 *
-	 * @param int $user_id User ID for Authorized User.
+	 * @param WP_REST_Request $request.
 	 */
-	public static function post_handler( $user_id ) {
-		$status = 200;
+	public static function post_handler( $request ) {
+		$user_id = get_current_user_id();
+		if ( ! $user_id ) {
```

Comment:
looks like this lost the earlier `is_wp_error` check? do we need to add it back here?

```php
if ( is_wp_error( $user_id ) ) {
	static::handle_authorize_error( 401, $user_id->get_error_message() );
}
```

Reply:
also, probably worth unifying with the same code in `query_handler`.

Reply:
oh, and we lost the code in `handle_authorize_error` that allows auth errors on localhost. that's useful. mind adding it back?

Reply:
ping on all three of these